package.json

```diff
@@ -1,25 +1,30 @@
 {
   "private": true,
   "description": "HASH monorepo",
-  "workspaces": [
-    "packages/blocks/*",
-    "packages/engine/apache-arrow-js-bundle",
-    "packages/hash/api",
-    "packages/hash/backend-utils",
-    "packages/hash/datastore",
-    "packages/hash/design-system",
-    "packages/hash/eslint-config",
-    "packages/hash/frontend",
-    "packages/hash/integration",
-    "packages/hash/playwright",
-    "packages/hash/realtime",
-    "packages/hash/search-loader",
-    "packages/hash/shared",
-    "packages/hash/task-executor",
-    "packages/hash/tsconfig",
-    "sites/hashai",
-    "sites/hashdev"
-  ],
+  "workspaces": {
+    "packages": [
+      "packages/blocks/*",
+      "packages/engine/apache-arrow-js-bundle",
+      "packages/hash/api",
+      "packages/hash/backend-utils",
+      "packages/hash/datastore",
+      "packages/hash/design-system",
+      "packages/hash/eslint-config",
+      "packages/hash/frontend",
+      "packages/hash/integration",
+      "packages/hash/playwright",
+      "packages/hash/realtime",
+      "packages/hash/search-loader",
+      "packages/hash/shared",
+      "packages/hash/task-executor",
+      "packages/hash/tsconfig",
+      "sites/hashai",
+      "sites/hashdev"
+    ],
+    "nohoist": [
+      "**/ts-invariant"
+    ]
```

Comment:
The new configuration of dependencies and sub-dependencies provided `@apollo/client` with a wrong version of `ts-invariant`, which caused the crash. Removing `nohoist` will be possible after upgrading `@apollo/client` and `apollo-server-express` and thus avoid incompatible `ts-invariant` subdependencies.

Some detail on the issue: https://github.com/apollographql/apollo-client/issues/9094#issuecomment-1176216035